stable/percona-xtradb-cluster/templates/statefulset.yaml

```diff
@@ -101,7 +101,8 @@ spec:
           timeoutSeconds: 2
         readinessProbe:
           exec:
-            command: ["mysql", "-h", "127.0.0.1", "-e", "SELECT 1"]
+            command:
+             ["mysql", "-h", "127.0.0.1", "-e", "SELECT 1"]
```

Comment:
This one should be:
```
command:
- mysql
- -h
- 127.0.0.1
- -e
- "SELECT 1"
```
if you would like to have multiline format; otherwise please keep as original. Detail see stable/percona/templates/deployment.yaml#L63